cc_licenses/templates/includes/legalcode_30_unported_license.html

```diff
@@ -5,7 +5,7 @@
 <div id="legal-code-body" class="padding-larger margin-top-bigger has-text-black">
   {% include 'includes/snippet/icon_header_snippet.html' %}
   <div>
-    <h3 class="padding-bottom-normal b-header">Creative Commons {{ license.title_english }}</h3>
```

Comment:
It looks like we are missing the `title` in the license header, side menu, and legal code section.

Reply:
I think running migrations and load_html_files again will resolve this, but let me know if not.